### ISSUE TITLE
Issue on Nano

### ISSUE THREAD

**Issue by andriyadi** [CongducPham/LowCostLoRaGw#7]
I've tried my own code and example code, tried to make sure the cabling is OK, there's an issue on Nano when executing this:
```
sx1272.sendPacketTimeout
```
The device always restarts. Weirdly, the message is managed to arrive on gateway. But I cannot do anything after calling that statement as the board restarts.

While the exact same code runs well on Uno.

Please advice. Thanks.

**Comment by CongducPham** [comment]
Be sure that you do not have memory issues. What code are you running on 
the Nano? A simple end-device code?

Le 12/04/16 00:50, Andri a écrit :
>
> I've tried my own code and example code, tried to make sure the 
> cabling is OK, there's an issue on Nano when executing this:
>
> |sx1272.sendPacketTimeout |
>
> The device always restarts. Weirdly, the message is managed to arrive 
> on gateway. But I cannot do anything after calling that statement as 
> the board restarts.
>
> While the exact same code runs well on Uno.
>
> Please advice. Thanks.
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly or view it on GitHub 
> <https://github.com/CongducPham/LowCostLoRaGw/issues/7>
>

C. Pham.


-- 
|------------ Congduc PHAM - Professor --------------------------|
|  LIUPPA - Equipe T2I              |         U.P.P.A. Pau       |
|  http://liuppa.univ-pau.fr/       |   http://www.univ-pau.fr   |
|----------------------------------------------------------------|
|UPPA, LIUPPA laboratory, UFR Sciences et Techniques             |
|Avenue de l'Université - BP 1155                                |
|64013 PAU CEDEX, FRANCE                                         |
|                                                                |
|phone:  [33] (0) 5 59 40 75 94                                  |
|fax:    [33] (0) 5 59 40 76 54                                  |
|Congduc.Pham@univ-pau.fr       http://www.univ-pau.fr/~cpham    |
|----------------------------------------------------------------|

**Comment by andriyadi** [comment]
Yep, the one that sends temperature. I even hardcode the temperature data, not actually accessing analogRead. Not much changes. 
Andri Yadi




On Tue, Apr 12, 2016 at 11:34 PM -0700, "Congduc Pham" <notifications@github.com> wrote:










Be sure that you do not have memory issues. What code are you running on 

the Nano? A simple end-device code?



Le 12/04/16 00:50, Andri a écrit :

>

> I've tried my own code and example code, tried to make sure the 

> cabling is OK, there's an issue on Nano when executing this:

>

> |sx1272.sendPacketTimeout |

>

> The device always restarts. Weirdly, the message is managed to arrive 

> on gateway. But I cannot do anything after calling that statement as 

> the board restarts.

>

> While the exact same code runs well on Uno.

>

> Please advice. Thanks.

>

> —

> You are receiving this because you are subscribed to this thread.

> Reply to this email directly or view it on GitHub 

> <https://github.com/CongducPham/LowCostLoRaGw/issues/7>

>



C. Pham.





-- 

|------------ Congduc PHAM - Professor --------------------------|

|  LIUPPA - Equipe T2I              |         U.P.P.A. Pau       |

|  http://liuppa.univ-pau.fr/       |   http://www.univ-pau.fr   |

|----------------------------------------------------------------|

|UPPA, LIUPPA laboratory, UFR Sciences et Techniques             |

|Avenue de l'Université - BP 1155                                |

|64013 PAU CEDEX, FRANCE                                         |

|                                                                |

|phone:  [33] (0) 5 59 40 75 94                                  |

|fax:    [33] (0) 5 59 40 76 54                                  |

|Congduc.Pham@univ-pau.fr       http://www.univ-pau.fr/~cpham    |

|----------------------------------------------------------------|
